actions/test/run_local.ts

```diff
@@ -235,7 +236,7 @@ async function processTx(
 ) {
   let hasErrors = false;
   const transaction = await provider.getTransaction(tx);
-  if (!transaction.blockNumber) {
+  if (!transaction?.blockNumber) {
     throw new Error(`The transaction ${tx} is not mined yet (no blockNumber)`);
   }
   const block = await provider.getBlock(transaction.blockNumber);
```

Comment:
I'll say this again.
Why do you need line 243?

It's already captured in line 239!

If you really want to check for the tx individually, move the check up

Reply:
👍